compiler/src/CMakeLists.txt

```diff
@@ -9,3 +9,4 @@ set(IREE_PACKAGE_ROOT_PREFIX "")
 set(IREE_COMPILER_TABLEGEN_INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}")
 
 add_subdirectory(openxla/compiler/nvgpu)
+add_subdirectory(openxla/compiler/async)
```

Comment:
Can you please move it under `nvgpu` folder (and `namespace openxla::compiler::nvgpu::async`), it's not quite accurate, but it will get it covered by CI jobs, and we'll find a better place later